src/inputs/select/select.jsx

```diff
@@ -42,6 +42,7 @@ const propTypes = {
     tabIndex: PropTypes.number,
     underline: PropTypes.bool,
     value: PropTypes.oneOfType([
+        PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
Is this an array of shapes when we're in multi-select mode?

Reply:
Yes sir.